Tests/UnitsTests/MeasurementTests.swift

```diff
@@ -210,6 +210,64 @@ final class MeasurementTests: XCTestCase {
         )
     }
 
+    func testPercent() throws {
+        XCTAssertEqual(
+            1.measured(in: .percent),
+            1.measured(in: .percent)
+        )
+
+        XCTAssertNotEqual(
+            100.measured(in: .percent),
+            1.measured(in: .none)
+        )
+
+        try {
+            let percent1 = 5.measured(in: .percent)
+            let percent2 = 5.measured(in: .percent)
+            XCTAssertEqual(
+                try percent1 + percent2,
+                10.measured(in: .percent),
+                accuracy: accuracy
+            )
+        }()
+
+        try {
+            let percent1 = 5.measured(in: .percent)
+            let value2 = 5.measured(in: .none)
+            XCTAssertThrowsError(try percent1 + value2)
```

Comment:
Ah, looks like this no longer fails, since I've added cross-unit addition/subtraction support since this MR was opened. This will no longer error and will result in `505%`.